src/EditorFeatures/Next/project.json

```diff
@@ -0,0 +1,8 @@
+﻿{
+  "dependencies": {
+    "Microsoft.VisualStudio.Language.Intellisense": "15.0.25123-Dev15Preview",
```

Comment:
I didn't make a dependencies project, because I hope this is the only place we'll put VS 15 editor dependencies.